### ISSUE TITLE
phpunit is failing

### ISSUE THREAD

**Issue by oliworx** [phpmath/vector#7]
using Ubuntu 16.04 / PHP 7.0 I can not run phpunit:

```
$ composer install --dev --prefer-source --no-interaction
You are using the deprecated option "dev". Dev packages are installed by default now.
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files

$ ./vendor/bin/phpunit
PHPUnit 4.7.2 by Sebastian Bergmann and contributors.
Warning:    No whitelist configured for code coverage

The Xdebug extension is not loaded. No code coverage will be generated.

PHP Fatal error:  Uncaught Error: Call to undefined function PHP\Math\BigNumber\bcadd() in /home/oli/projects/vector/vendor/phpmath/bignumber/src/BigNumber.php:305
Stack trace:
#0 /home/oli/projects/vector/vendor/phpmath/bignumber/src/BigNumber.php(46): PHP\Math\BigNumber\BigNumber->internalSetValue(1)
...
```

**Comment by waltertamboer** [comment]
Do you have the BCMath extension enabled? http://php.net/manual/en/book.bc.php

Travis CI does not fail: https://travis-ci.org/phpmath/vector/jobs/137184024

**Comment by waltertamboer** [comment]
I see that the phpmath/bignumber library does not have a requirement on ext-bcmath. I'll add it asap and than I'll update the reference to this library.

**Comment by oliworx** [comment]
Thanks, that was the missing point. I had to run `sudo apt install php7.0-bcmath` and now phpunit is running fine. 
Maybe we can add some install istructions to the README.

**Comment by waltertamboer** [comment]
I'm all up for making usage easier end users. The only challenge I see is that you can install PHP in many different ways (package repository, from source, various tools like phpbrew, etc.) It's a bit beyond the scope of this package to show one how to install the requirements.

Once the composer file of phpmath/bignumber is updated with a requirement to `ext-bcmath` things become a lot clearer already. In that case you would have been notified that you were missing an extension.

What do you think?

**Comment by oliworx** [comment]
This will probably be sufficient

**Comment by waltertamboer** [comment]
Ok, than I'll close this issue for now. Thanks!

**Comment by waltertamboer** [comment]
Fixed in commit [d87b2bdd9583f22313c7f239d8b503448a7a0c58](https://github.com/phpmath/vector/commit/d87b2bdd9583f22313c7f239d8b503448a7a0c58)
